main/hunspell/APKBUILD

```diff
@@ -1,21 +1,18 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=hunspell
-pkgver=1.6.1
+pkgver=1.6.2
 pkgrel=0
 pkgdesc="Spell checker and morphological analyzer library and program"
-url="http://hunspell.github.io/"
+url="https://hunspell.github.io/"
 arch="all"
 license="GPL2+ LGPL2+ MPL 1.1"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-depends=""
 makedepends="gettext-dev ncurses-dev readline-dev autoconf automake libtool"
-source="$pkgname-$pkgver.tar.gz::https://github.com/hunspell/hunspell/archive/v$pkgver.tar.gz
-	"
+source="$pkgname-$pkgver.tar.gz::https://github.com/hunspell/hunspell/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
 
-builddir="$srcdir"/$pkgname-$pkgver
 prepare() {
 	cd "$builddir"
-	default_prepare
```

Comment:
Any particular reason why you are removing this?

Reply:
I tested without it and it builds fine. I think the smaller an APKBUILD the better.
But if it's needed I can add it back.

Reply:
Yes, it builds. But if a patch is ever added to `$sources` it will not be applied without this. Generally we've wanted to keep `default_prepare` so we don't get patches left out accidentally if they are added.

Reply:
Good to know.
I fixed it.